projects/lights_control/test/test_lights_blinker.c

```diff
@@ -114,9 +114,8 @@ void test_lights_blinker_deactivate_already_inactive(void) {
   TEST_ASSERT_OK(lights_blinker_init(&blinker, TEST_LIGHTS_BLINKER_DURATION_SHORT));
   TEST_ASSERT_OK(lights_blinker_activate(&blinker, LIGHTS_EVENT_GPIO_PERIPHERAL_SIGNAL_LEFT));
   TEST_ASSERT_OK(lights_blinker_deactivate(&blinker));
-  TEST_ASSERT_NOT_OK(lights_blinker_deactivate(&blinker));
-  Status s = status_get();
-  TEST_ASSERT_EQUAL(s.message, "Blinker already inactive");
+  StatusCode s = lights_blinker_deactivate(&blinker);
+  TEST_ASSERT_EQUAL(s, STATUS_CODE_INVALID_ARGS);
```

Comment:
The expected syntax for unity asserts is `TEST_ASSERT_[x](expected, actual)`.

Reply:
😬 whoops...